packages/plugin-list/test/List.test.ts

```diff
@@ -0,0 +1,2410 @@
+import { expect } from 'chai';
```

Comment:
@Goaman I need your mocha expertise for this commit: the tests are not being concatenated with the other tests.